server/poetry.lock

```diff
@@ -759,6 +759,23 @@ files = [
 [package.dependencies]
 langchain-core = ">=0.3.66,<0.4.0"
 
+[[package]]
+name = "enthusiast-file-parsers"
```

Comment:
So if the server can't run without it, why is it a separate package?

Reply:
It can run without it. You are right, I've added handling missing parser exception

Reply:
But then the file uploads won't work, right?